cheroot/test/test_conn.py

```diff
@@ -8,8 +8,12 @@
 import six
 import pytest
 
-from cheroot._compat import HTTPConnection, HTTPSConnection, NotConnected, BadStatusLine
-from cheroot._compat import ntob, urlopen
+from cheroot._compat import HTTPConnection
```

Comment:
1. Imports should usually [be on separate lines](https://www.python.org/dev/peps/pep-0008/#imports).
2. The length of the line 11 [exceeds 79 characters](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).